OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.Validations.Add.cs

```diff
@@ -42,8 +42,13 @@ public async void ShouldThrowValidationExceptionOnAddWhenExamAttachmentIsNullAnd
                 broker.InsertExamAttachmentAsync(It.IsAny<ExamAttachment>()),
                     Times.Never);
 
+            this.dateTimeBrokerMock.Verify(broker =>
```

Comment:
You don't need this line here. but definitely the one below it the `VerifyNoOtherCalls` is definitely correct.